openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -72,12 +72,12 @@ def report_completion(
     # List record count per media type
     media_type_reports = ""
     for media_type, counts in record_counts_by_media_type.items():
-        if not counts.upserted:
+        if counts is None or not counts.upserted:
```

Comment:
I think we may also want to add some test cases for when `counts` is `None` - let me know if you'd like some advice or assistance on that!

Reply:
Would you mind taking a look at [test_reporting.py](https://github.com/WordPress/openverse-catalog/pull/492/files#diff-bcf55852f44e45efd0d72a9721e230594272b44290c0923e531705e29e751f97)? I think it was easier than I thought, but let me know if you were expecting something really different!

Reply:
That's perfect! I think this PR is ready 😄